### PR TITLE
Legal revisions

### PR DESCRIPTION
**Pull request by dgkutnic**

### CODE REVIEW

File: docs/docs/edsl.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 nav_order: 5
 ---
 
-# C++ Tile eDSL
+# C++\* Tile eDSL
```
Comment (flaub) [inline docs/docs/edsl.md]: Seriously? Let's tone this way down to just entities that actually have lawyers.

**Comment by flaub** [comment]
How was it that this was all ok when it was `.md` files sitting in the `docs` dir but now all the sudden it's not OK?

File: docs/index.md [hunk 2]
```diff
@@ -13,17 +13,17 @@ hardware is not well supported or the available software stack contains
 unpalatable license restrictions.
 
 PlaidML sits underneath common machine learning frameworks, enabling users to
-access any hardware supported by PlaidML. PlaidML supports [Keras], [ONNX], and
+access any hardware supported by PlaidML. PlaidML supports [Keras\*], [ONNX\*], and
```
Comment (flaub) [inline docs/index.md]: not a trademark

File: docs/index.md [hunk 3]
```diff
@@ -13,17 +13,17 @@ hardware is not well supported or the available software stack contains
...
 [nGraph].
 
 As a component within the [nGraph Compiler stack], PlaidML further extends the
 capabilities of specialized deep-learning hardware (especially GPUs,) and makes
 it both easier and faster to access or make use of subgraph-level optimizations
 that would otherwise be bounded by the compute limitations of the device.
 
-As a component under [Keras], PlaidML can accelerate training workloads with
+As a component under [Keras\*], PlaidML can accelerate training workloads with
```
Comment (flaub) [inline docs/index.md]: not a trademark

File: docs/index.md [hunk 4]
```diff
@@ -33,7 +33,7 @@ Clover, check out the instructions at [building PlaidML](docs/building.md) to
 build a custom configuration to support your hardware.
 
 ## Prerequisites
-- Python (v2 supported, v3 recommended)
+- Python\* (v2 supported, v3 recommended)
```
Comment (flaub) [inline docs/index.md]: not a trademark

File: docs/index.md [hunk 5]
```diff
@@ -167,10 +167,10 @@ correctness as part of our continuous integration system.
   * examples/imdb_lstm.py (from keras)
 
 [nGraph Compiler stack]: https://ngraph.nervanasys.com/docs/latest/
-[Keras]: https://keras.io/
-[GitHub]: https://github.com/plaidml/plaidml/issues
+[Keras\*]: https://keras.io/
```
Comment (flaub) [inline docs/index.md]: not in a link

File: docs/index.md [hunk 6]
```diff
@@ -167,10 +167,10 @@ correctness as part of our continuous integration system.
...
+[GitHub\*]: https://github.com/plaidml/plaidml/issues
```
Comment (flaub) [inline docs/index.md]: not in a link

File: docs/index.md [hunk 7]
```diff
@@ -167,10 +167,10 @@ correctness as part of our continuous integration system.
...
 [plaidml-dev]: https://groups.google.com/forum/#!forum/plaidml-dev
-[Onnx]: https://github.com/onnx
+[Onnx\*]: https://github.com/onnx
 [nGraph]: https://github.com/NervanaSystems/ngraph
-[slack workspace (#plaidml)]: https://join.slack.com/t/ngraph/shared_invite/enQtNjY1Njk4OTczMzEyLWIyZjZkMDNiNzJlYWQ3MGIyZTg2NjRkODAyYWZlZWY5MmRiODdlNzVkMjcxNjNmNWEyZjNkMDVhMTgwY2IzOWQ
+[Slack\* workspace (#plaidml)]: https://join.slack.com/t/ngraph/shared_invite/enQtNjY1Njk4OTczMzEyLWIyZjZkMDNiNzJlYWQ3MGIyZTg2NjRkODAyYWZlZWY5MmRiODdlNzVkMjcxNjNmNWEyZjNkMDVhMTgwY2IzOWQ
```
Comment (flaub) [inline docs/index.md]: not in a link

File: docs/index.md [hunk 8]
```diff
@@ -167,10 +167,10 @@ correctness as part of our continuous integration system.
   * examples/imdb_lstm.py (from keras)
 
 [nGraph Compiler stack]: https://ngraph.nervanasys.com/docs/latest/
-[Keras]: https://keras.io/
-[GitHub]: https://github.com/plaidml/plaidml/issues
+[Keras\*]: https://keras.io/
+[GitHub\*]: https://github.com/plaidml/plaidml/issues
 [plaidml-dev]: https://groups.google.com/forum/#!forum/plaidml-dev
-[Onnx]: https://github.com/onnx
+[Onnx\*]: https://github.com/onnx
```
Comment (flaub) [inline docs/index.md]: not in a link

**Comment by flaub** [comment]
Also, we should use superscripted asterisks so it's a lot less painful to the eyes

File: docs/_config.yml [hunk 9]
```diff
@@ -54,6 +54,7 @@ logo: /assets/images/plaid-final.png
 
 # Footer content appears at the bottom of every page's main content
 # footer_content: "Copyright &copy; 2017-2019 Patrick Marsceill. Distributed by an <a href=\"https://github.com/pmarsceill/just-the-docs/tree/master/LICENSE.txt\">MIT license.</a>"
+footer_content: "&copy; Copyright Intel Corporation | <a href=\"https://www.intel.com/content/www/us/en/legal/trademarks.html\">*Trademarks</a> | <a href=\"https://www.intel.com/content/www/us/en/legal/terms-of-use.html\">Terms of Use</a> | <a href=\"https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html\">Privacy</a> | <a href=\"https://www.intel.com/content/www/us/en/privacy/intel-cookie-notice.html\">Cookies</a>"
```
Comment (flaub) [inline docs/_config.yml]: Let's put this in a `footer.html`, this seems way too much for a one-liner

**Comment by dgkutnic** [comment]
We had asterisks in place in our `.md` files before, they just became inconsistent as content was added and refactored. We could also revert the ones that were removed when the website was created.

More importantly, we were missing the trademark disclosure. Asterisks mean absolutely nothing if you don't actually signify what they mean.

I think the best course of action is probably to send our docs to an actual TM&B person who can make all the edits. I'm obviously not a legal expert and I was just doing this based off my experience making webinars. I don't think it's the best use of my time to individually Google each of these and read the trademark usage policy to determine whether they need an asterisk or not. 

-----

BTW, python is a registered trademark of the PSF: https://www.python.org/psf/trademarks/

**Pull request closed without merging**